platforms/common/glfwApp.cpp

```diff
@@ -397,8 +404,10 @@ void keyCallback(GLFWwindow* window, int key, int scancode, int action, int mods
 
 void dropCallback(GLFWwindow* window, int count, const char** paths) {
 
+    const char* apiKey = std::getenv("MAPZEN_API_KEY");
```

Comment:
This value is already accessed once at setup, we can change its scope to be global to this file.